dlp/snippets/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-dlp-samples</artifactId>
+  <version>0.0.1-SNAPSHOT</version><!-- This artifact should not be released -->
+  <packaging>pom</packaging>
+  <name>Google Cloud Data Loss Prevention Samples Parent</name>
+  <url>https://github.com/googleapis/java-dlp</url>
+  <description>
+    Java idiomatic client for Google Cloud Platform services.
+  </description>
```

Comment:
Update here as well like [this](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/dataproc/pom.xml#L4-L8)

Reply:
Updated.